lib/debezium/converters/geometry.go

```diff
@@ -69,8 +69,14 @@ func (GeometryPoint) Convert(value any) (any, error) {
 	return string(bytes), nil
 }
 
-// ParseGeometry will take in an object with b64 encoded wkb and return a GeoJSON string.
-func ParseGeometry(value any) (string, error) {
+type Geography struct{}
```

Comment:
Is there a reason for calling this `Geography` instead of `Geometry` like it was before?

Reply:
Haha yeah I named it Geography because we already have a variable named Geometry

Reply:
And this parser handles both Geography + Geometry

Reply:
I could rename [Geometry](https://github.com/artie-labs/transfer/blob/875987a24aa5e4248ba4704c39583c56203ad6b6/lib/debezium/converters/geometry.go#L23) to be GeometryJSON.

Wdyt?

Reply:
Ah okay, yeah I think that seems more consistent/user-friendly to me (Geometry -> GeometryJSON and Geography -> Geometry)